src/lib.rs

```diff
@@ -383,6 +375,18 @@ macro_rules! add_type_bounds {
     }
 }
 
+#[doc(hidden)]
+#[macro_export]
+macro_rules! display_message_fun_for_struct {
```

Comment:
I think you forgot to remove this macro.

Reply:
thanks :-)